### PR TITLE
Create the August 2019 meeting notes

### PR DESCRIPTION
**Pull request by 1Copenut**
Adding notes from VSP monthly call with the VA 508 office

### CODE REVIEW

File: platform/accessibility/meeting-notes/2019-08.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	* **Feedback Item 3:** Will issue a ticket to remove the error messages on field blur. Accessibility more than compliance. Mark as lower priority.
+	* **Feedback Item 4:** This is a known issue, logged as a [forms fix](https://github.com/department-of-veterans-affairs/va.gov-team/issues/314) in VA.gov-team
+	* IMPROVE: The data entry type (in 10% increments, and instructions) should be presented on the page instead of in error handling.
+1. VSP discovered a bug in our forms system where checkboxes and radio groups will not read the `<legend>` if screen reader users are navigating with the `TAB` key. Turns out the legned must be the first child of the fieldset, or it won't work correctly and throws a Best Practices warning in axe. Logged as a [sitewide improvement](https://github.com/department-of-veterans-affairs/va.gov-team/issues/767) in VA.gov-team repo.
+1. The in-form "Application has been saved" message we discussed in April is in design discussion: [Application saved message could be missed by assistive devices](https://github.com/department-of-veterans-affairs/va.gov-team/issues/182)
```
Comment (peterhill-oddball) [inline platform/accessibility/meeting-notes/2019-08.md]: Used (1) twice, but markdown doesn't care and just keeps numbering.  I'm not sure what MD best practice is.

File: platform/accessibility/meeting-notes/2019-08.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+2. Facility Locator has a number of issues from the third-party audit in Triage. This is our team that reviews, scopes, and distributes issues to be fixed.
+3. Accessibility vs. Compliance - WCAG Error Reporting, in the spec. Look into this and further clarify it. 
+
+
+## Upcoming Quick Tests
+1. Education STEM Scholarship feature - Internal audit done, will send for Quick Test as soon as final issues are fixed.
+1. Outreach and Events- Internal audit done, will send for Quick Test as soon as final issues are fixed.
```
Comment (peterhill-oddball) [inline platform/accessibility/meeting-notes/2019-08.md]: Used (1) twice, but markdown doesn't care and just keeps numbering. I'm not sure what MD best practice is.

File: platform/accessibility/meeting-notes/2019-08.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+2. VET TEC updates to GI Bill Comparison Tool®. Internal audit starting today.
+
+
+## Local Issue Tracking
+1. GI Bill Comparison Tool - 2 issue opens
```
Comment (peterhill-oddball) [inline platform/accessibility/meeting-notes/2019-08.md]: "2 issue opens" == "2 open issues" ?

**Review by pjhill — APPROVED** [review]
I nitpicked some stuff, but I don't see any real issues with anything described in the meeting notes.

**Pull request merged**